### PR TITLE
Object Storage Controls Revision

### PR DESCRIPTION
**Pull request by eddie-knight**
Collaboration with @mlysaght2017 and @kazmik23 as part of [Milestone 6](https://github.com/finos/common-cloud-controls/milestone/6)

### CODE REVIEW

**Comment by mlysaght2017** [comment]
@eddie-knight @kazmik23 - added in fixes, and addressed TODOs - please review. Added in some new TODOs, mainly related to thinking about lifting some of the objstor specific controls to common controls

**Review by mlysaght2017 — APPROVED** [review]

File: services/storage/object/features.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ common_features:
   - CCC.F05 # Signed URLs
   - CCC.F06 # Identity Based Access Control
   - CCC.F07 # Event Notifications
+  - CCC.F21 # Data Replication
```
Comment (smendis-scottlogic) [inline services/storage/object/features.yaml:9]: There should be a common-feature.yaml entry for CCC.F21

File: services/common-controls.yaml [hunk 2]
```diff
@@ -156,3 +153,39 @@ controls:
...
+      - CCC.TH09
+    nist_csf: DE.AE-1
+    test_requirements:
+      tlp_green:
+        01: |
+          Simulate non-human enumeration activities (e.g., with service accounts) and verify that alerts are triggered in the logging system.
+        02: |
+          Confirm that logs are properly generated and accessible for review following non-human enumeration attempts.
+
+  - id: CCC.ObjStor.C08 # Enable multi-zone or multi-region data replication
```
Comment (sshiells-scottlogic) [inline services/common-controls.yaml:174]: This has an inconsistent format compared to the other ids (CCC.CXY)

**Review by sshiells-scottlogic — APPROVED** [review]

File: services/common-controls.yaml (hunk 2 above)
Comment (eddie-knight) [reply services/common-controls.yaml:174]: Oof! I'll fix that in the follow-up. Thank you.

**Pull request merged**